docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -258,3 +258,20 @@ Limitations
   query with a partition key as a filter.
 * ``IN`` list filters are only allowed on index (that is, partition key or clustering key) columns.
 * Range (``<`` or ``>`` and ``BETWEEN``) filters can be applied only to the partition keys.
+
+.. _cassandra-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
+the Cassandra database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
```

Comment:
Additionally, Cassandra connector supports DELETE statement with partition columns filter.

Reply:
Can you provide more details on this? Does that mean that the connector _only_ supports DELETE with a PARTITION BY statement?

Reply:
To my untrained eye, that sounds like a more granular read on the DELETE WHERE limitation we describe in [this fragment](https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector/sql-delete-limitation.fragment). Do you think this would be appropriate to tack on here, or does this connector need more finely crafted language?

Reply:
We'll need a new DELETE fragment, then.

Reply:
Fragment would be overkill IMO. These are the only two connectors (Hive and Cassandra) with this limitation and there aren't going to be more of these.

Also the text in Hive cannot be used as a fragment because it's part of a larger section - not standalone.

IMO just add something like:

```
`DELETE` is only supported if the WHERE clause matches entire partitions.
```

Reply:
+1 to the above suggestion, just add it under a new SQL DELETE sub-section below SQL support.